src/components/alarm/AlarmController.cpp

```diff
@@ -0,0 +1,118 @@
+//
+// Created by mrussell on 30.08.21.
+//
+// Copied from Florian's Timer app
```

Comment:
Not the correct format for a license header.

Reply:
@Avamander Could you advise a better header? Probably something similar to https://github.com/JF002/InfiniTime/blob/develop/src/displayapp/screens/Music.cpp#L1

Reply:
Yes, that would be consistent with the rest.

Reply:
Added license header to new source files and deleted original header. I changed the copyright date to 2021.

Reply:
Great improvement, but a few minor issues.